examples/pylab_examples/stock_demo.py

```diff
@@ -1,19 +1,51 @@
 import matplotlib.pyplot as plt
 import numpy as np
 
+import matplotlib.cbook as cbook
 from matplotlib.ticker import MultipleLocator
-from data_helper import get_two_stock_data
+
+"""
```

Comment:
You can put the example's docstring at the beginning of the file; also, add the titling format as [indicated here](http://matplotlib.org/devdocs/devel/MEP/MEP12.html#clean-up-guidelines).